tests/integration/test_compatibility.py

```diff
@@ -158,6 +158,29 @@ def test_order_by_no_limit():
     )
 
 
+@pytest.mark.skip(reason="WIP DataFusion")
```

Comment:
Jeremy had added this test but is not complete. The SQL references tables that do not exists. I am not sure if we should just remove this test from the PR?

Reply:
Yeah we should be good to remove this test and just verify that https://github.com/dask-contrib/dask-sql/issues/725 is working as expected